api/__init__.py

```diff
@@ -0,0 +1,54 @@
+import base64
+
+from flask import Flask, request
+import numpy as np
+
+
+def create_app():
+    app = Flask(__name__)
+
+    @app.route('/')
+    def routes():
+        return {
+            "spectrum": "/spectrum",
+            "mean": "/mean",
+            "median": "/median",
+        }
+
+    @app.route('/spectrum', methods=['POST', 'GET'])
```

Comment:
Route / method comments are needed hear to explain what /spectrum is mean to produce.  Some of the variables below are not used in the response.